src/FastRoute/src/RcTreeBuilder.cpp

```diff
@@ -51,10 +51,10 @@ using std::abs;
 using std::min;
 
 RcTreeBuilder::RcTreeBuilder(ord::OpenRoad* openroad,
-			     DBWrapper* dbWrapper,
+			     GlobalRouter* fr,
 			     Grid* grid)
 {
-  _dbWrapper = dbWrapper;
+  _fr = fr;
```

Comment:
the variable shouldn't be called fr anymore

Reply:
Renamed all fr vars to grouter